modules/nf-core/prokka/main.nf

```diff
@@ -31,18 +31,26 @@ process PROKKA {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args   ?: ''
-    prefix   = task.ext.prefix ?: "${meta.id}"
-    def proteins_opt = proteins ? "--proteins ${proteins[0]}" : ""
-    def prodigal_tf = prodigal_tf ? "--prodigaltf ${prodigal_tf[0]}" : ""
+    def args             = task.ext.args   ?: ''
+    prefix               = task.ext.prefix ?: "${meta.id}"
+    def fasta_compressed = fasta.getExtension() == "gz" ? true : false
```

Comment:
`? true : false` seems redundant. Was there a specific reason for implementing it this way?

Reply:
I think while I was prototyping I had some lines that were very long - but I've tidied this up now,